crypto/ed25519-donna/README.md

```diff
@@ -1,10 +1,10 @@
-[ed25519](http://ed25519.cr.yp.to/) is an 
-[Elliptic Curve Digital Signature Algortithm](http://en.wikipedia.org/wiki/Elliptic_Curve_DSA), 
-developed by [Dan Bernstein](http://cr.yp.to/djb.html), 
-[Niels Duif](http://www.nielsduif.nl/), 
-[Tanja Lange](http://hyperelliptic.org/tanja), 
-[Peter Schwabe](http://www.cryptojedi.org/users/peter/), 
-and [Bo-Yin Yang](http://www.iis.sinica.edu.tw/pages/byyang/).
+[ed25519](https://ed25519.cr.yp.to) is an 
```

Comment:
Please send the changes to upstream project as well: https://github.com/floodyberry/ed25519-donna